### DISCUSSION TITLE
Data Importer - Auto import fails cannot connect to local instance

### DISCUSSION THREAD

**Discussion by Plutoaurus** [firefly-iii/firefly-iii#7854] [Firefly III] [answered]
I'm almost there with setting everything up!
Running really smoothly so far.

Just following the guide to get the auto import working with nordigen. I've enabled the env settings.

When I run the curl command I get the error:
Could not connect to your local instance.

I have tried setting the env to a few things inc:
http://192.168.0.10/firefly-iii/public
http://192.168.0.10/firefly-iii/public/
http://127.0.0.1/firefly-iii/public
http://localhost/firefly-iii/public
(First two = not actual ip, on a different subnet behind a firewall incase anyone says I'm mad :))

It could be in the curl command the authorisation header is wrong? Wasn't 100% what this should be.

Non docker install, same host for firefly and importer.
Manual imports run from my laptop(different IP) work

Thanks!!

P.s. love the comment writing style!

**Comment by JC5** [comment]
Did you press Reauthenticate from the user interface whenever you change those values?

**Reply by Plutoaurus** [reply]
I have now.
No longer asks for details just authenticates itself through the Web gui.

**Reply by JC5** [reply]
Great!

**Reply by Plutoaurus** [reply]
So the curl fails on line 69 of autouploadcontroller.php

The Haveaccess() bit so it does seem like it's connecting but not allowed.

In that authorisation bit, what is the format?
I have accou t with email, password, and 2fa

**Reply by JC5** [reply]
It needs an access token. Does that help?

**Reply by Plutoaurus** [reply, marked as answer]
Found it!
Access token has to be in
" "
Woo.

Now to find why it imported duplicates. To the .json file I go :)
Edit. Must have been a old file, replaced and all is good.
Tags are the best for oopsies!
